plugins.d/nvm.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+if [ -d "~/.nvm" ]; then
+
+  echo "🔀 Upgrading NVM (node version manager"
+  export CURRENT_PWD=`pwd`
+
```

Comment:
I think you forgot to actually `cd` into the nvm dir.

Also try using `pushd ~/.nvm > /dev/null` and `popd > /dev/null` instead of (re)storing the current `pwd`.

Reply:
Yep

Reply:
Also don't hard code `~/.nvm` use $NVM_DIR